<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>648</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Краткая памятка</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>190</y>
      <width>461</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>148</red>
             <green>186</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>138</red>
             <green>165</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>колесо для подбора палитр</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>148</red>
             <green>186</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>138</red>
             <green>165</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ещё одно колесо для подбора</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>148</red>
             <green>186</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>240</red>
             <green>240</green>
             <blue>240</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>138</red>
             <green>165</green>
             <blue>215</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>191</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="LinkVisited">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>247</red>
             <green>190</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>библиотека готовых палитр</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>540</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
